src/java.desktop/share/classes/javax/swing/JInternalFrame.java

```diff
@@ -188,6 +188,8 @@
     protected String  title;
     /**
      * The icon that is displayed when this internal frame is iconified.
+     * Subclassers must ensure this is set to a non-null value
+     * during construction and nor subsequently set this to null.
```

Comment:
nor->not (looks like a typo I made in the CSR) as r and t keys are adjacent. I fixed the CSR but you'll have to to fix it here, otherwise seems fine